sale_order_secondary_unit/models/sale_order.py

```diff
@@ -63,3 +63,6 @@ def onchange_product_uom_for_secondary(self):
     @api.onchange('product_id')
     def onchange_secondary_unit_product_id(self):
         self.secondary_uom_id = self.product_id.sale_secondary_uom_id
+        if self.secondary_uom_id:
+            self.secondary_uom_qty = 1.0
```

Comment:
Shouldn't this be run after quantity set and put the proper amount after conversion of the quantity to this secondary one?

Reply:
This is only when the product is changed as a default value